catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/SolrCache.java

```diff
@@ -391,7 +391,7 @@ public UpdateResponse delete(String query) throws IOException, SolrServerExcepti
         @Override
         protected SolrQuery postAdapt(QueryRequest request, SolrFilterDelegate filterDelegate,
                 SolrQuery query) throws UnsupportedQueryException {
-            if (!request.isEnterprise()) {
+            if (!request.isEnterprise() && request.getSourceIds() != null) {
```

Comment:
@coyotesqrl Why does it no longer matter if the request is enterprise or not anymore?